components/Comment/Comment.js

```diff
@@ -31,22 +34,24 @@ export default function Comment({ comment, mutateComments }) {
   };
 
   return (
-    <article>
+    <li>
       <CommentContainer>
         <ProfilePicture
-          src={userImageURL}
+          src={createdBy.image}
           alt="profile picture"
           height={36}
           width={36}
         />
         <CommentText>
           <CommentHeader>
-            {userName}{" "}
+            {createdBy.name}{" "}
             <CommentTime> · {creationDate && timeElapsed}</CommentTime>
           </CommentHeader>
           <CommentBody>{text}</CommentBody>
           <FlexContainer>
-            <ReplyButton onClick={handleReplyForm} text="Antworten" />
+            {session ? (
+              <ReplyButton onClick={handleReplyForm} text="Antworten" />
+            ) : null}
```

Comment:
If we don't want to show anything if the session is falsey, then we can use the `&&` operator. 

```suggestion
            {session && (
              <ReplyButton onClick={handleReplyForm} text="Antworten" />
            )}
```